src/main.cpp

```diff
@@ -609,6 +631,12 @@ int main ( int argc, char** argv )
                 CInstPictures::UpdateTableOnLanguageChange();
             }
 
+            std::unique_ptr<CClientRpc> pClientRpc;
```

Comment:
Similar comment to the above.  I'm guessing it's something to do with thread management.